frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/simulated-region-popup/simulated-region-popup.component.html

```diff
@@ -8,7 +8,7 @@ <h5 class="popover-header">
             data-cy="simulatedRegionPopupCloseButton"
         ></button>
     </h5>
-    <div class="popover-body" style="width: 500px; height: 300px">
+    <div class="popover-body" style="width: 500px; height: 350px">
         <ng-container *ngIf="(currentRole$ | async) === 'trainer'">
             <app-simulated-region-overview-general
```

Comment:
If many behaviors are added to a simulated region, the list might be larger than the popup height, so we should enable scrolling.

A quick fix would be to add the following classes to `app-simulated-region-overview-general`: `class="d-block mh-100 overflow-auto"`. This would cause the whole popup contents to scroll if they're larger than 350px. The downside of this approach is that the tabs themselves would be scrolled out of view, too. This does not look that nice, but I did not manage to keep them in place and restrict the size of the tab content only.

Reply:
done